tests/conftest.py

```diff
@@ -367,6 +363,20 @@
 ]
 
 
+@contextmanager
```

Comment:
Is this used automatically for each test?

Reply:
no, need to set explicitly for every test.
probably will apply later to specific tests.

just want to complete this pr